infrastructure/modules/ses/main.tf

```diff
@@ -1,49 +1,29 @@
 resource "aws_ses_domain_identity" "ndr_ses" {
-  domain = "${terraform.workspace}.${var.domain}"
+  domain = var.domain
```

Comment:
was this intended?

Reply:
yep, `domain` is the required keyword for setting up an ses identity and `var.domain` is a modified domain string set in infrastructure/ses.tf